pkg/cli/aws/client.go

```diff
@@ -58,13 +63,19 @@ func (c *client) GetCallerIdentity(ctx context.Context, region string, accessKey
 }
 
 // ListRegions lists the AWS regions available (fetched from EC2.DescribeRegions API).
-func (c *client) ListRegions(ctx context.Context, region string, accessKeyID string, secretAccessKey string) (*ec2.DescribeRegionsOutput, error) {
-	credentialsProvider := credentials.NewStaticCredentialsProvider(accessKeyID, secretAccessKey, "")
-	ec2Client := ec2.New(ec2.Options{
-		Region:      region,
-		Credentials: credentialsProvider,
-	})
-	result, err := ec2Client.DescribeRegions(ctx, &ec2.DescribeRegionsInput{})
+func (c *client) ListRegions(ctx context.Context, region string) (*ec2.DescribeRegionsOutput, error) {
+	// Load the AWS SDK config and credentials
+	cfg, err := config.LoadDefaultConfig(ctx, config.WithSharedConfigProfile("default"))
```

Comment:
Is `config.WithSharedConfigProfile("default")` for default config as well? Doesn't `LoadDefaultConfig` do that automatically? https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#loading-aws-shared-configuration

Reply:
thanks for pointing this, removed config with sharedProfile